code/game/mecha/combat/five_stars.dm

```diff
@@ -1,22 +1,41 @@
-/obj/mecha/combat/five_stars
-	desc = "A state of the art tank deployed by the Spinward Stellar Coalition National Guard."
-	name = "\improper Tank"
+/obj/mecha/combat/combat_tank
+	name = "\improper  Armored Combat Vehicle \"Tank\""
+	desc = "While exosuits have taken over armored vehicles in peformance in ground combat, poorer factions have to rely on these old relics. The 207 was exclusively mass produced for a Sarthini war, before a abrupt surrender, leaving many of these to gain dust in warehouses."
```

Comment:
Sarthini?

Reply:
desc = "While exosuits have taken over armored vehicles in peformance in ground combat, factions that can not afford expensive exosuits have to rely on these old relics. The 207 was exclusively mass produced for a Kalixcis war, however a abrupt surrender left many of these to gain dust in warehouses."
is this better?

Reply:
Kalixcian?

Reply:
an abrupt surrender

Reply:
desc = "While exosuits have taken over armored vehicles in performance in ground combat, factions that can not afford expensive exosuits have to rely on these old relics. The 207 was exclusively mass produced for a Kalixcian war, however an abrupt surrender left many of these to gain dust in warehouses."